packages/fuselage/src/components/SideBarV2/SideBarItem/SideBarItemAction.tsx

```diff
@@ -0,0 +1,26 @@
+import type { HTMLAttributes } from 'react';
+import React from 'react';
+
+import { Icon } from '../../Icon';
+
+export const SideBarItemAction = ({
+  children,
+  className,
+  ...props
+}: { onClick: (e: Event) => void } & HTMLAttributes<HTMLDivElement>) => (
+  <li>
+    <div
+      role='button'
+      className={[
+        'rcx-sidebar-v2-item rcx-sidebar-v2-item__action',
+        className && className,
+      ]
+        .filter(Boolean)
+        .join(' ')}
+      {...props}
+    >
+      <Icon name='plus-small' size='x20' />
```

Comment:
Will always be this icon?

Reply:
i'll make it default `plus-small`, but changeable. you never know :)